docs/start/tutorial.md

```diff
@@ -1487,8 +1487,18 @@ You could certainly do this as a controlled component, but you'll end up with mo
 
 Notice how controlling the input requires three points of synchronization now instead of just one. The behavior is identical but the code is more complex.
 
-```jsx filename=src/routes/root.jsx lines=[1,6,9-11,25-28]
+```jsx filename=src/routes/root.jsx lines=[1,6-7,16,19-21,35-38]
 import { useEffect, useState } from "react";
+// existing code
+
+ export async function loader({ request }) {
+  const url = new URL(request.url);
+  let q = url.searchParams.get("q");
+  if (!q) { q = ''}
```

Comment:
the main change is in these 2 lines where instead of `undefined` if `q` is falsy, we set the input value on a component mount to an empty string